Sources/BottomSheet/Core/Presentation/BottomSheetPresentationController.swift

```diff
@@ -150,7 +150,7 @@ public final class BottomSheetPresentationController: UIPresentationController {
     }
 
     public override var shouldPresentInFullscreen: Bool {
-        false
+        true
```

Comment:
Should we set it only for IOS 17?

I'm not sure how this flag affects older versions
```Suggestion
        if #available(iOS 17.0, *) {
            return true
        } else {
            return false
        }
```